README.md

```diff
@@ -1,54 +1,45 @@
-# The Style Guide
-
-1. Define meaningful types for every variant and polymorphic variant you introduce. For example, when defining `MergeBranch` you would create `type branchName = string;` and the definition would look like: `MergeBranch(branchName)` instead of `MergeBranch(string)`
-2. Define type signatures explicitly unless you're sure that they should not be defined. Even when the compiler can infer the type it still is often useful to state it explicitly for readability of the code, especially outside of the IDE.
-3. Use named parameters if a function has multiple parameters of same type. This will prevent accidentally mixing up the parameters when calling the function.
-4. Tend to use named parameters more often. This is advised for readability.
-5. Imperative code (i.e. the `ref` keyword) should be used only with a very strong reason.
-6. Default to using non polymorphic variants (```type rgb = Red | Green | Blue;``` over ```type rgb = [`Red | `Green | `Blue];```). We think of polymorphic variants as a lesser-typed approach and prefer to have stronger typing.
-7. Prefer using uncurried functions over curried (i. e. *`reduceU` over `reduce`* in Belt.List*)*, it will help the compiler to produce cleaner error messages in cases when there is a parameters mismatch, which happens quite often during refactoring. Another benefit is that it works faster.
-8. All new components should be Jsx3 components. This is done to be able to use the latest language features.
-9. Put multiline styles in the top of the file. 
-10. When deciding between `thing->function->function` and `function(thing)->function` read it out and pick the one that makes more sense in a natural language, i.e. `Firebase.firestore(firebase)->Firebase.root("main")` and `maybeSomething->Belt.Option.getWithDefault("definitelySomething")`
-11. Avoid switch-like funs 
-
-    ```jsx
-    ~~let map = f =>
-      fun
-      | Some(v) => Some(f(v))
-      | None => None;~~
-    ```
-  This syntax is hard to parse and it is deprecated in latest versions of the language.
-12. When working with JSON create decoders and encoders with the `bs-json` lib
+# The Avo Rescript Style Guide
+
+At [Avo](https://github.com/avohq) we use Rescript as our primary programming language. To allow the development team to deliver better code and reviews, we've come up with a set of guidelines:
```

Comment:
Should we use avo.app link instead of github?

Reply:
I'm up for that!
```suggestion
At [Avo](https://www.avo.app/) we use Rescript as our primary programming language. To allow the development team to deliver better code and reviews, we've come up with a set of guidelines:
```